### DISCUSSION TITLE
Models UI “Input Lag” with Enabling/Disabling

### DISCUSSION THREAD

**Discussion by clduab11** [open-webui/open-webui#7841] [Issues]
Hi all!

I wanted to second a FR that was posted in Discussions that allows all new models to be DISABLED by default. Even though I’d have liked to not open a new Discussion, nothing I’ve seen from a cursory perspective indicates the issue I’m seeing.

As it stands, I’m working with 153 models. 5-6 local, the rest via API call. When a new API is hooked into OWUI, or a new Function is added as a Manifold…the Models are automatically enabled by default. When I call a new API (Cohere, in my case) via the Cohere manifold @ OWUI’s site…it introduces about 20 models into the screen. I have a video that I can’t shrink down anymore, but will try to upload via Discord that shows the below…

what’s happening is the UI lags enough to where when I hit the toggle, it’ll clearly Enable/Disable the model, but occasionally, the toggle switches itself back. 

Mass-toggling Enable/Disable models a few at a time becomes a painful experience when you have to scroll around to go back and re-toggle and then wait, hoping the “input lag” from clicking the toggle doesn’t make all your disabling/enabling efforts in vain.

Is this something in the works to be fixed?

I have a video displaying exactly what’s going on, but is too large of a video to upload here. If I see my entry hit the Discord, I’ll try to upload it there instead.

**Comment by asbrwl** [comment]
I am seeing the exact same experience as @clduab11.
The only way it's working for me at present is to toggle a model and wait for 30 seconds before toggling another. I've got the network tab open in dev tools, waiting till I get the success on the toggle request.
